modules/rostests/apitests/ws2_32/recv.c

```diff
@@ -113,6 +113,21 @@ int Test_recv()
     ok(readIosb.Information == RECV_BUF, "Short read\n");
 
     NtClose(readEvent);
+
+    /* Now keep reading until the end (CORE-17425) */
+    int ret;
```

Comment:
You could resuse the existing `iResult` variable (that is not used as well anymore after that in the function).